### PR TITLE
v1.3.0 release

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,16 @@
 sudo: false
+
 language: node_js
+
 node_js:
   - "0.10"
```
Comment (leobalter) [inline .travis.yml]: should we still keep 0.10?

**Comment by leobalter** [comment]
assuming we're keeping 0.10 both on travis and appveyor config files, this LGTM

File: .travis.yml (hunk 1 above)
Comment (vladikoff) [inline .travis.yml]: Yea i think we never said "grunt drops 0.10 support" so we kept this, unless we really need to drop it.

Comment (leobalter) [inline .travis.yml]: It works for now, so why not?

**Pull request merged**